alarm/nialarm.yml

```diff
@@ -2,7 +2,7 @@ swagger: '2.0'
 info:
   version: '1'
   title: Alarm Web Service
-  description: SystemLink Alarm Service HTTP API
+  description: SystemLink Alarm Service HTTP API. Only available on SystemLink Server.
```

Comment:
My plan is to update the rest of the docs with this verbage in a separate PR if Mark and Caley are happy with it.

Reply:
I like it.  👍